extensions-core/lookups-cached-global/src/main/java/io/druid/query/lookup/NamespaceLookupExtractorFactory.java

```diff
@@ -95,7 +96,12 @@ public NamespaceLookupExtractorFactory(
   public boolean start()
```

Comment:
yes, sorry, I mean that it does not provide an easy way for callers to determine if they were interrupted or if this is a plain vanilla (possibly retryable) exception. Maybe that's okay, it depends on whether callers need to be able to figure that out or not.